### DISCUSSION TITLE
MAC System blocks the installation.

### DISCUSSION THREAD

**Discussion by momo1986** [LorenEteval/Furious#118] [Q&A] [answered]
<img width="422" alt="屏幕快照 2024-10-30 下午8 11 29" src="https://github.com/user-attachments/assets/c7cb8e13-b69c-46fd-bea2-ac483d725c08">
SOS, I cannot run it.

Thanks & Regards!

**Comment by vint2k** [comment, marked as answer]
Hi
try to run in terminal: `xattr -dr com.apple.quarantine /Applications/Furious-GUI.app`
